nuxt.config.js

```diff
@@ -54,6 +54,17 @@ export default {
       { hid: 'twitter:image', property: 'twitter:image', content: site.image },
     ],
     link: [{ rel: 'icon', type: 'image/x-icon', href: '/favicon.svg' }],
+    script: [
+      {
+        hid: 'btw-widget', async: true, defer: true, id: 'btw-widget',
+        'data-project-code': '54bb5a67-baf9-4df7-93d4-f90ec061e166',
```

Comment:
Should we move the project code and events secret to env variables? If we are making that changes, please update the `.env.example` file and add the keys on our password manager.

Reply:
I guess that's a good idea, will do.